microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -55,39 +106,47 @@ def _prepare_data_disk(
         node.execute(f"mkdir {mount_point}", sudo=True)
 
 
-def _get_smb_version(node: Node) -> str:
-    if node.tools[KernelConfig].is_enabled("CONFIG_CIFS_SMB311"):
-        version = "3.1.1"
-    else:
-        version = "3.0"
-    return version
-
-
-def _prepare_azure_file_share(
+# Updates as of march 2025.
+# Default premium SKU will be used for file share creation.
+# This will ensure SMB multi channel is enabled by default
+def _deploy_azure_file_share(
     node: Node,
-    account_credential: Dict[str, str],
-    test_folders_share_dict: Dict[str, str],
-    fstab_info: str,
-) -> None:
-    folder_path = node.get_pure_path("/etc/smbcredentials")
-    if node.shell.exists(folder_path):
-        node.execute(f"rm -rf {folder_path}", sudo=True)
-    node.shell.mkdir(folder_path)
-    file_path = node.get_pure_path("/etc/smbcredentials/lisa.cred")
-    echo = node.tools[Echo]
-    username = account_credential["account_name"]
-    password = account_credential["account_key"]
-    echo.write_to_file(f"username={username}", file_path, sudo=True, append=True)
-    echo.write_to_file(f"password={password}", file_path, sudo=True, append=True)
-    node.execute("cp -f /etc/fstab /etc/fstab_cifs", sudo=True)
-    for folder_name, share in test_folders_share_dict.items():
-        node.execute(f"mkdir {folder_name}", sudo=True)
-        echo.write_to_file(
-            f"{share} {folder_name} cifs {fstab_info}",
-            node.get_pure_path("/etc/fstab"),
-            sudo=True,
-            append=True,
+    environment: Environment,
+    names: List[str],
```

Comment:
Sorry, I just found actually it needs key/value pairs. So, 

1. pass in the pairs to the method like `names: dict[str, str]`.
2. And then pass into `azure_file_share.create_file_share` like `file_share_names=names.values()`
3. Set test_folders_share_dict.
   ```python
   test_folders_share_dict : Dict[str, str] = {}
   for key, value in names:
        test_folders_share_dict[key] = fs_url_dict[value]
   ```